main.py

```diff
@@ -122,17 +122,27 @@ def cal_dividend_increament(div_obj: pd.core.series.Series, number_of_years: int
                    'is_monotonic': is_monotonic,
                    'is_persistent': is_persistent}
     return output_dict
-# %%
 
-dividend_aristocrats = ['DOV','GPC','PG','EMR','MMM','CINF','KO','JNJ','CL','ITW','HRL','SWK','FRT','SYY','GWW','BDX','PPG','TGT','ABBV','ABT','KMB','PEP','NUE','SPGI','ADM','WMT','VFC','ED','LOW','ADP','WBA','PNR','MCD','MDT','SHW','BEN','APD','AMCR','XOM','AFL','CTAS','ATO','MKC','TROW','CAH','CLX','CVX','AOS','ECL','WST','ROP','LIN','CAT','CB','EXPD','BRO','ALB','ESS','O','IBM','NEE','CHD','GD']
-ido_list = ['JNJ', 'XOM', 'CVX', 'KO', 'MCD', 'RTX', 'IBM', 'ADP', 'TGT', 'ITW', 'CL', 'APD', 'EMR', 'AFL', 'ED', 'WBA', 'GPC', 'CLX', 'FC', 'PII', 'SON', 'LEG', 'MGEE', 'WLYB', 'UVV', 'TDS', 'ARTNA', 'MMM']
-dividaat_list = ['ALB','BANF','BEN','CAH','CARR','CB','CBSH','CBU','CHRW','ES','GPC','KTB','LANC','LECO','MO','PB','RBCAA','SCL','SWK','TROW','UGI','UMBF','VFC']
-defence_companies_list = ['LMT', 'RTX', 'ESLT', 'BA', 'GD', 'NOC', 'BAESY', 'EADSY', 'THLEF', 'SAIC','HII','LHX','GE','HON','LDOS','HII','TDG','TXT']
-indexes = ['SCHD', 'VIG', 'VYM', 'VNQ','VNQI','RWO','MORT','REZ']
+def import_settings() -> dict:
+    with open("./settings.json") as settings_file:
+        settings = json.load(settings_file)
+    return settings
+
+def import_ticker_list(settings) -> list:
+    with open("./inputs/tickers.json") as ticker_file:
+        tickers_dict = json.load(ticker_file)
 
-tickers = list( set(dividend_aristocrats).union( set(ido_list), set(dividaat_list), set(defence_companies_list), set(indexes) ) )
+    included_ticker_lists = settings["included_ticker_lists"]
```

Comment:
every time you access a field in the settings you should safely check if it exists

in the life of the program new settings are added, and because the run_settings.json file is not versioned, a user may have an older file and may not have one of the fields accessed in the program

another way to handle this run_settings.json file is to add a "version" field. every time a new field is added the version is incremented
when you open the file:
- if the version is up to date, you can access all fields safely
- if the version is not up to date, create a new run_settings.json file and copy previous data to it